core/domain/fs_services.py

```diff
@@ -21,6 +21,13 @@
 
 
 def get_exploration_file_system_class():
+    """Returns ExplorationFileSystem class to the client if the development mode
```

Comment:
the development mode of constants is on --> DEV_MODE is True.

Reply:
Done